src/main/webapp/app/components/timeSeries/TimeSeries.tsx

```diff
@@ -96,7 +96,7 @@ const TimeSeriesInfo = ({ date, children }: ITimeSeriesInfoProps) => {
 
   useEffect(() => {
     function updateLineHeight() {
-      setLineHeight(timelineRef.current.clientHeight - lastChildRef.current.clientHeight - 10); // 10 is height of margin from react-event-timeline
+      setLineHeight((timelineRef.current?.clientHeight ?? 0) - (lastChildRef.current?.clientHeight ?? 0) - 10); // 10 is height of margin from react-event-timeline
```

Comment:
Can we set the height to null when the ref is undefined. Setting it to -10 seems like it would cause a bug

Reply:
can do